docs/website/data/integrations.yaml

```diff
@@ -30,6 +30,20 @@ integrations:
     code:
       - https://github.com/dapr/dapr
 
+  fig:
+    title: fig
+    description: Beatiful shell autocompletion for OPA and many other commands, for Mac OS
+    software:
+      - fig
+    labels:
+      category: utilities
+      layer: shell
+    code:
+      - https://github.com/open-policy-agent/contrib/tree/main/opa_fig_autocomplete
+      - https://github.com/withfig/autocomplete/blob/master/src/opa.ts
+    inventors:
+      - fig
```

Comment:
"Inventor" feels a little weird here. Is it required?

Reply:
It's a pretty weird wording for _any_ integration IMHO.. I read it more like "the organization behind the project or product"